ProjectLighthouse/Extensions/RequestExtensions.cs

```diff
@@ -15,7 +15,21 @@ namespace LBPUnion.ProjectLighthouse.Extensions;
 
 public static class RequestExtensions
 {
-
+    static RequestExtensions()
+    {
+        Uri captchaUri = ServerConfiguration.Instance.Captcha.Type switch
+        {
+            CaptchaType.HCaptcha => new Uri("https://hcaptcha.com"),
+            CaptchaType.ReCaptcha => new Uri("https://www.google.com/recaptcha/api/"),
```

Comment:
Does it matter that one has a trailing slash and the other doesn't?

Reply:
In testing it does not.

Reply:
Should be fine then.

Reply:
Alright, merging then.